tests/ui/test_settings_enable_2fa.py

```diff
@@ -0,0 +1,43 @@
+import re
+
+from playwright.sync_api import Page, expect
+from tenacity import RetryCallState, retry, stop_after_attempt, wait_random_exponential
+
+BASE_URL = "http://127.0.0.1:8080"
```

Comment:
This URL (or rather, port) should probably come via a fixture of some sort using a trick like this:

https://github.com/freedomofpress/securedrop/blob/6ad1fa95330c4f36c3144cb871620c64d8bc1f94/securedrop/tests/functional/conftest.py#L43-L48

I'm guessing the UI tests require an already running app that's started in a second terminal or so, yeah?